FLIR/conservator/cli/managers.py

```diff
@@ -98,6 +99,38 @@ def count(search_text):
 
     if issubclass(type_manager, CollectionManager):
 
+        @group.command(help="Create a new collection with the given path.")
+        @click.argument("collection_path")
+        @click.option(
+            "-r", "--recursive", is_flag=True, help="Create all missing path components"
```

Comment:
When would someone not want to use `-r`?

Reply:
I'd suggest renaming to `-p, --parents` "make parent directories as needed" to match `mkdir`

Reply:
> When would someone not want to use `-r`?

I think it is useful to avoid creating paths by mistake in case of typos, when the parent path is expected to exist.  Especially when run by a user with permission to create new projects.

I like the comparison with mkdir, '-r' is now '-p'.